submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,68 @@
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/contacts.png" alt=""></div><p>Contact</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/tanslation.png" alt=""></div><p>Translate</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/videos.png" alt=""></div><p>Video</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/search.png" alt=""></div><p>Search</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/photos.png" alt=""></div><p>Photo</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/apps.png" alt=""></div><p>Apps</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/documents.png" alt=""></div><p>Document</p></div></a>               
+                            </div>
+                            <!-- <button class="buttom" ><a href="#openModal2"> More</a></button> -->
+                            <input class="secondInput" type="checkbox"  id="second1-navbar-button" />
+                            <label for="second1-navbar-button" class="second-navbar-button">                                 
+                                <div class="buttomPoupApp"><p>More</p></div>
+                            </label>
+                        <!-- <a href="#openModal" class="buttomPoupApp"><div class="buttomPoupApp"><p>More</p></div></a>    -->
+                         <div class="line"></div>
+                             <div class="appSecond" id="openModal">
+                           
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/chat.png" alt=""></div><p>Calendar</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/classroom.png" alt=""></div><p>Mail</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/collection.png" alt=""></div><p>News</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/mars.png" alt=""></div><p>Map</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/blogs.png" alt=""></div><p>Files</p></div></a>
+        <a class="buttonApp" href=""><div class="itemsApp"><div class="imgBloc"><img src="Image/promo.png" alt=""></div><p>Contact</p></div></a>                              
+                                  
+        <p class="text_end">Even more from Cattans</p>
+    </div>
+                                
+    </div>
+    </div>
+</body>
+</html>
```

Comment:
could you please add an empty line

Reply:
Good practice - naming of selectors should be in one style (underscore or hyphen or camelCase)